tracer/src/Datadog.Trace/AppSec/WafEncoding/Encoder.cs

```diff
@@ -483,11 +517,7 @@ void EnumerateAndEncodeIList(IList lstInstance)
                 }
 
                 default:
-                    if (Log.IsEnabled(Vendors.Serilog.Events.LogEventLevel.Debug))
-                    {
-                        Log.Debug("Couldn't encode object of unknown type {Type}, falling back to ToString", o.GetType());
-                    }
-
+                    Log.Warning("Couldn't encode object of unknown type {Type}, falling back to ToString", o.GetType());
```

Comment:
I don't know if we should make this a warning - we saw a _lot_ of these being logged before, and it's another case of "if it happens once, it is probably going to happen a _lot_"